source/mainnet/net/release-notes/release-notes-lp.rst

```diff
@@ -167,12 +167,22 @@ Wallets
 |bw|
 -------------------------
 
-    January 10, 2024
+    January 29, 2024
 
-    The EuroE token is now added to all accounts by default.
+    Version 1.3.2 includes several changes and bug fixes.
+
+    The CIS-2 token lookup no longer blocks choosing a contract if looking up metadata or balance only fails for some tokens.
+    The missing date for delegation/validation stake decrease/stop has been restored.
+    Changing restake preference is no longer blocked when the validator is below minimum stake threshold.
+    `SendTransaction` in wallet-api now supports `bigint` as part of smart contract parameters, fixing an issue with using large numbers.
+    And users can view their secret recovery phrase in the wallet settings if they need to write it down again.
```

Comment:
Two more  bugs were fixed:

1. BW crashes if in the stake field value is larger than 64 bit int while updating the stake
2. Inject script loads wasm and inject on all pages